listener.go

```diff
@@ -114,9 +121,19 @@ func processChangeList(changes *common.ChangeList) {
 			case common.Insert:
 				ds := makeDataScopeFromRow(change.NewRow)
 				err = insertDataScope(ds, tx)
+
```

Comment:
@sramamoorthy73 There's no race, cause the only routine reading/writing the cache here is `datascopeCacheManager`. Sync is done by chan instead of locks.